advertisers/models.py

```diff
@@ -0,0 +1,75 @@
+import os
+
+from django_extensions.db.models import TimeStampedModel
+
+from django.db import models
+from django.utils.translation import gettext as _
+
+from accounts.models import UserProfile
+
+
+class Advertiser(TimeStampedModel):
+    """
+    Advertiser model
+    """
+
+    email = models.EmailField(blank=False, null=False)
+    advertisement = models.ForeignKey(
+        "advertisers.Advertisement",
+        on_delete=models.SET_NULL,
+        related_name="advertiser",
+        blank=True,
+        null=True,
+    )
+    monthly_budget = models.IntegerField(blank=False, null=False)
```

Comment:
@autn for now let's make the monthly budget linked to an advertisement, not the advertiser.